src/fieldset/index.jsx

```diff
@@ -62,7 +61,7 @@ class Fieldset extends Component {
       <fieldset>
         {
           map(schema, ({ inputType, label, conditionalReveal, showIf, accessor, format, ...props }, key) => {
```

Comment:
Should prolly remove accessor here as well?

Reply:
yes, you are right! hmm not sure why lint didn't pick that up